sale_payment_mgmt/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,17 @@
+This addon adds a Payments (only customer payments) menu in Sales to users are in some of this Sale groups:
+    User: Own Documents Only
+    User: All Documents
+    Administrator
+
+.. image:: ../static/description/payment_menu.png
```

Comment:
You can draw an arrow pointing to the menu entry.

Reply:
Image updated now